test/lib/eventually.hh

```diff
@@ -15,8 +15,16 @@
 
 #include "seastarx.hh"
 
+using sleep_fn = std::function<future<>(std::chrono::milliseconds)>;
+
+struct seastar_sleep_fn {
+    future<> operator()(std::chrono::milliseconds ms) const {
+        return seastar::sleep(ms);
+    }
+};
```

Comment:
This could be just a simple function too.

Reply:
okay.  Will also move it out of line.